nbs/04a_export.ipynb

```diff
@@ -133,7 +133,7 @@
    "outputs": [],
```

Comment:
<p>Carefuly of python mutable parameters!</p><p>https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/928/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>